django_filters/filters.py

```diff
@@ -362,7 +362,7 @@ def get_max_validator(self):
         """
         Return a MaxValueValidator for the field, or None to disable.
         """
-        return MaxValueValidator(1e50)
+        return MaxValueValidator(int(1e50))
```

Comment:
I just want to have a look at the order of validation in the field, but maybe this should be a decimal already... 
(i.e. did we already run the value from the widget through `to_python`, giving us a decimal in the `value < limit` check...)